ctms/models.py

```diff
@@ -93,7 +93,11 @@ def primary_email_insensitive_comparator(cls):
     # Indexes
     __table_args__ = (
         Index("bulk_read_index", "update_timestamp", "email_id"),
-        Index("idx_email_primary_email_lower", func.lower(primary_email)),
+        Index(
+            "idx_email_primary_unique_email_lower",
+            func.lower(primary_email),
+            unique=True,
+        ),
```

Comment:
In this same way that this index "covers" `idx_email_primary_email_lower`, is there also a plain unique index on `primary_email` via the `unique` flag set in the [model](https://github.com/mozilla-it/ctms-api/blob/2201ec68cc0c0a84ffdda84184bf130bd839cb37/ctms/models.py#L60)? If not, maybe we can set that flag to `False`, since I think that's essentially there to generate a unique index.

Reply:
As discussed elsewhere:

> I would say that this index does not hurt. Only drawback of indexes is cost of write and db/ram size. I don't think we are the level of optimizing these